samples/event_manager_profiler/README.rst

```diff
@@ -0,0 +1,87 @@
+.. _event_manager_profiling_sample:
+
+Event Manager profiling tracer
+##############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Event Manager profiling tracer sample demonstrates the functionality of profiling :ref:`event_manager` events using the :ref:`profiler` and the :ref:`event_manager_profiler_tracer` modules.
+
+The sample application consists of the following modules that communicate using events:
+
+* Module a (:file:`module_a.c`) - This module waits for a configuration event sent by :file:`main.c`.
+  After receiving this event, it starts sending periodic events every second to module b.
+  The module also has a counter which increases before the one second event is sent and the value is transmitted during the event.
+  When the module receives a five second event from module b, it zeros out the counter, sends a burst of 50 burst events and simulates work for 100ms by busy waiting.
+* Module b (:file:`module_b.c`) - This module waits for one second events from module a.
+  Every time a one second event is received, the module checks if the value it transmits is equal to 5.
+  If it is equal to 5, then module b sends a five second event.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf9160dk_nrf9160_ns, nrf52dk_nrf52832, nrf52840dk_nrf52840, nrf21540dk_nrf52840
+
+Configuration
+*************
+
+|config|
+
+Building and running
+********************
+.. |sample path| replace:: :file:`samples/event_manager_profiler`
+
+.. include:: /includes/build_and_run.txt
+
+Testing
+=======
+
+After programming the sample to your development kit, test it by performing the following steps:
+
+1. Run the script :file:`real_time_plot.py` located in the :file:`scripts/profiler` folder, with the name that is to be used to store the data as argument.
+   For example, run the following command to generate a :file:`test_name.csv` and a :file:`test_name.json` file using the *test_name* argument:
+
+   .. parsed-literal::
+      :class: highlight
+
+      real_time_plot.py *test_name*
+
+   The script opens a GUI window that displays events as points on timelines, similar to the following:
+
+   .. figure:: /images/event_manager_profiling_sample.png
```

Comment:
The image has a black line at the bottom. Please redo the framing to remove this line.